src/main/java/org/carlmontrobotics/subsystems/Limelight.java

```diff
@@ -89,16 +103,59 @@ public double getDistanceToNoteMeters() {
     }
   }
 
-  public double getArmAngleToShootSpeakerRad(){
+  public double getArmAngleToShootSpeakerRad() {
     double armRestingHeightToSubwooferMeters = HEIGHT_FROM_RESTING_ARM_TO_SPEAKER_METERS;
     double horizontalDistanceMeters = getDistanceToSpeakerMeters() + SIDEWAYS_OFFSET_TO_OUTTAKE_MOUTH;
     return END_EFFECTOR_BASE_ANGLE_RADS - Math.atan(armRestingHeightToSubwooferMeters / horizontalDistanceMeters);
   }
 
-  public double getRotateAngleDeg() {
+  public double getRotateAngleRad() {
     double cameraLensHorizontalOffset = getTXDeg(SHOOTER_LL_NAME) / getDistanceToSpeakerMeters();
     double realHorizontalOffset = Math.atan(cameraLensHorizontalOffset / getDistanceToSpeakerMeters());
     return Math.atan(realHorizontalOffset / getDistanceToSpeakerMeters());
   }
```

Comment:
I'm not sure if this falls under the scope of this PR, but something's weird with this math.
``` Java
double cameraLensHorizontalOffset = Deg / Meters;
double realHorizontalOffset = atan(deg/m^2); // Arguments passed to atan should almost always be unitless
return atan(rad/m); // Arguments passed to atan should almost always be unitless
```
If this is something that would be better addressed in a different PR, feel free to ignore this comment, and I can just open an issue.

Reply:
if megatags work though, you don't have to do all this math. I'd say we see if megatags work and if so, we can probably scrap the math.